tests/abi/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+add_library(abi_compat_common INTERFACE)
```

Comment:
Please document the goal of the test cases - either here or inside the sources.

Reply:
Will do. I've already heavily restructured things and am still working on the interop test case.